### ISSUE TITLE
Implement new design for User Input Settings

### ISSUE THREAD

**Issue by kuasha420** [google/site-kit-wp#5896]
## Feature Description

The User Input Settings section of the Admin Settings should be updated to reflect the updated look. 

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* The User Input section of Admin Settings page should implement the new [design](https://www.figma.com/file/CdtrHB0HAjfGRWTnTMMQI0/SummaryWidget?node-id=866%3A22969). 
    * The layout, typography, positioning of various UI elements and text copies should exactly follow the design.
* Note that the settings section will only be visible when the user has completed the User Input questions. Otherwise, they will see the CTA that's being implemented in #5895. 
    * Ensure that the logic is correct and either the CTA or the question is displayed according to the User Input completion state. 
* The Key Metrics toggle should not be implemented as part of this issue. 
* The questions will be updated separately as part of #5888. 
* The Inline Edit interface will be implemented in #5897. 

## Implementation Brief

- In `assets/js/components/settings/SettingsAdmin.js`:
    - Update the `<Layout />` `title` to be `Key metrics` matching the Figma designs.
	- Add the `Edit your answers for more personalized metrics:` sub-heading matching the Figma designs, before `<UserInputPreview />` is rendered.
    - Make sure the sub-heading and questions only show up if at least one question is answered (you may be able to use logic introduced in #5900). If not answered, show the CTA from #5895.
- In `assets/js/components/user-input/UserInputPreview.js`:
    - Once #5891 is merged, make sure the new `Review your answers` sub-heading can be conditionally rendered, using a prop like the existing `noFooter`, say `noHeading`. This is because we don't want to show this in the Settings Page (`assets/js/components/settings/SettingsAdmin.js`).

### Test Coverage

- No tests need to be added/updated.

## QA Brief

- Ensure that the `userInput` feature flag is enabled in the tester plugin.
- Set up the Site Kit plugin (do not answer all the user input questions yet, so that you can see the CTA in Admin Settings).
- Go to Site Kit -> Settings -> Admin Settings.
- Verify that the settings section for user input is now named `Key metrics`.
- Navigate to the `/wp-admin/admin.php?page=googlesitekit-user-input` URL.
- Answer the user input questions.
- Verify that the Admin Settings screen now shows the User Input section according to the Figma design linked in the ACs.
- Verify that the concerns raised [here](https://github.com/google/site-kit-wp/issues/5896#issuecomment-1378776259) have been addressed (except 1, since that is using the correct token).
- Verify that the second concern raised [here](https://github.com/google/site-kit-wp/issues/5896#issuecomment-1381692646) is resolved.

## Changelog entry

* Update styling of User Input Settings on the Admin Settings tab.

**Comment by aaemnnosttv** [comment]
@kuasha420 what is implemented as part of this issue? It seems like it would only be the "Key metrics" container with nothing in it? If so we can probably handle this as part of #5895 

Otherwise, I think this seems like it would be a condition to render either the CTA or the settings, both of which are implemented in other issues.

**Comment by kuasha420** [comment]
I think we could've implemnted CTA and this one together, however the other one was moved far up into IB and is with @hussain-t.

I've updated the AC of this one to add note about the CTA. This should be good now. 

This has turned into kind of a glue issue. If, during the IB process, there turns out to be nothing to do here, we can close it then. However, still worth keeping it open for now and see if everything fits together and follows the design and intent.

**Comment by hussain-t** [comment]
@kuasha420, I agree; we could've combined both into a single task. However, I have updated the IB for #5895. So the CTA container can be implemented as part of this ticket.

**Comment by tofumatt** [comment]
I get what you mean about this mostly being a "glue" issue @kuasha420, which I think made these ACs a bit unclear to me—they mostly seem like what will be implemented elsewhere/what _not_ to implement.

Could you clarify exactly what this issue is adding/being changed by this issue in one section of the ACs? And then if you want to clarify as a separate point what's covered elsewhere by other issues that's fine, but I think it's currently mixed which is hard to parse 🙂

**Comment by kuasha420** [comment]
Fair point @tofumatt ! I have reordered the ACs to put what needs to be done in the beginning and hopefully it reads clearer. I've also mentioned showing the CTA or the questions correctly based on the completion state. 

As for that the issue is mainly addressing, it's the aforementioned logic and general design update ie. layout, font, copy etc. \

Cheers.

**Comment by tofumatt** [comment]
Thanks @kuasha420, that makes sense!

In that case ACs seem good. 👍🏻

**Comment by nfmohit** [comment]
@kuasha420 I'm working on the IB for this one and I have a question. In the "Answers editing" part of the Figma designs, the new inline edit interface introduces two new elements which I think aren't being implemented in #5897:
1. Other authors of the answer to the question.
2. Saving the question inline.

Should those be introduced as a part of this issue?

CC: @aaemnnosttv @tofumatt

**Comment by eugene-manuilov** [comment]
IB ✔️

**Comment by derweili** [comment]
Unassigned me, since I did not start execution and got many other issues back from CR, that I have to work on.

**Comment by mohitwp** [comment]
### QA Update ❌ 

@nfmohit 

1) Background color of selected answer is different. In Figma color is #B8E5CA and implemented color is #b0e6c5.

<details>

![image](https://user-images.githubusercontent.com/94359491/211816181-513c0c07-8a76-4b51-aabe-fd399501e83e.png)

![image](https://user-images.githubusercontent.com/94359491/211816411-7364a73a-62ec-4ba2-8b5e-3cd57b07d41d.png)

</details>

2) On edit answer screen. Options are not aligned as Figma design.

<details>

Figma -

![image](https://user-images.githubusercontent.com/94359491/211816766-640ad3ce-2d16-4af9-a981-24fbda0ae3bd.png)

![image](https://user-images.githubusercontent.com/94359491/211816992-519c90fd-d11b-453b-8717-68a353215781.png)

</details>

3) Questions title font color in Figma is #202124 and implemented color is #131418 and Font Family in Figma is 'Google Sans Text' and implemented is 'Google Sans Display'.

<details>

![image](https://user-images.githubusercontent.com/94359491/211817356-1f8d3a9a-a864-43b9-9ff5-eef619b266ce.png)

![image](https://user-images.githubusercontent.com/94359491/211817408-380d68e0-e73e-4ae4-8deb-e19d343a8a20.png)

![image](https://user-images.githubusercontent.com/94359491/211821223-b17b1660-2a0a-4818-80a1-c8d86321b52e.png)

![image](https://user-images.githubusercontent.com/94359491/211821566-970eab6a-0832-4f1d-b2be-2ed040a458b7.png)

</details>

4) Answers Loading Issue > When user navigate to admin settings then for milliseconds "Please select an answer" text gets load before actual answer.

<details>

https://user-images.githubusercontent.com/94359491/211818274-e2a12059-a3db-49c4-a72c-9497e80e7032.mp4

</details>

5) On edit answer, screen separation line not exists between options in Figma design.

<details>

Figma 

![image](https://user-images.githubusercontent.com/94359491/211818512-4acbf625-274b-4414-9063-881a9ff0b149.png)

![image](https://user-images.githubusercontent.com/94359491/211818777-5271264f-78b8-4ef5-82e8-82698bde8488.png)

</details>

**Comment by nfmohit** [comment]
Thank you for raising the concerns, @mohitwp!

I have addressed all of them, except #1 (`Background color of selected answer is different. In Figma color is #B8E5CA and implemented color is #b0e6c5`), this has been implemented using the correct token/variable, and will be fixed via #6328.

Although, I feel most of these concerns are related to #5891 (re-design of the User Input Preview component), since we re-use the Preview component in the Admin Settings screen. This issue was defined mostly to address the container of the Preview component in the Admin Settings screen. Regardless, I've addressed the concerns as a part of this issue.

Thanks!

**Comment by mohitwp** [comment]
### QA Update ⚠️ 

@nfmohit I found 2 issues. Let me know if you would resolve the below issue in the same ticket, or I should create a separate ticket for this ?

Issue 1 : Loading issue> If analytics is connected then for milliseconds "CTA gets load" before we see the answer preview screen.

<details>


https://user-images.githubusercontent.com/94359491/212305531-0eec3d22-f951-4a3a-adc9-7b7c15ef5357.mp4

</details>

2) Now under options alignment for radio buttons is same as Figma design but check boxes in 3rd question are still not aligned.

<details>

![image](https://user-images.githubusercontent.com/94359491/212305987-d26d59bf-1a79-4af6-a03f-231c71bdd799.png)

![image](https://user-images.githubusercontent.com/94359491/212306059-b721da95-d09d-4e92-adc9-0f1c2755496f.png)

</details>

**Comment by nfmohit** [comment]
Thank you for re-checking this, @mohitwp!

> Issue 1 : Loading issue> If analytics is connected then for milliseconds "CTA gets load" before we see the answer preview screen.

Since it doesn't directly relate to the design in question, I think this should be addressed in a new issue.

> Now under options alignment for radio buttons is same as Figma design but check boxes in 3rd question are still not aligned.

I have added a new PR #6403 for this.

Thanks!

**Comment by kuasha420** [comment]
@mohitwp once you create the follow up issues, please assign them to me for triage. Cheers.

**Comment by mohitwp** [comment]
@kuasha420 Here is the follow-up issue - https://github.com/google/site-kit-wp/issues/6428 .
